src/config.civet

```diff
@@ -0,0 +1,17 @@
+fs from node:fs
+
+export type Config = {
+	token: string;
+	clientID: string;
+	admins: string[];
+}
+
+export default function config(): Config
+	try
+		return JSON.parse fs.readFileSync("./config.json", "utf8")
+	catch
+		return {
+			token: "",
+			clientID: "",
+			admins: []
+		}

```

Comment:
I can't imagine this default being that useful, as it lacks a token... wouldn't it be better to let the error throw (i.e. remove the `throw` block), so the user gets feedback about what's going wrong (e.g. JSON parse error)?